### PR TITLE
Chore: header responsiveness

### PR DESCRIPTION
**Pull request by imriven**
Responsive Header

- media queries for cell, tablet, desktop, XL desktop
- navbar component to display at tablet size 1000px or more
- links to components

https://trello.com/c/itQkEK96/60-task-navbar-responsiveness

### CODE REVIEW

**Comment by imriven** [comment]
[Task: NavBar Responsiveness](https://trello.com/c/itQkEK96/60-task-navbar-responsiveness)

**Comment by imriven** [comment]
The navbar could still use some styling to show you which link you're on.

**Comment by imriven** [comment]
[Chore: Add home to hamburger menu & logo](https://trello.com/c/alrhUDvX/59-chore-add-home-to-hamburger-menu-logo)

**Review by chelsabeth — APPROVED** [review]
I like all your breakpoints and thank you for adding the home navigation to the logo 👍 Looks good!

File: src/components/common/AppHeader.js [hunk 1]
```diff
@@ -10,16 +12,60 @@ const StyledHeader = styled(Header)`
...
   justify-content: space-between;
   align-items: center;
-  margin: 5% 2%;
   padding-bottom: 3%;
   border-bottom: 1px solid grey;
-  height: 55px;
+  width: 100%;
+  left: 0;
+  background: rgba(255, 255, 255, 0.8);
+  z-index: 1;
```
Comment (vtopham) [inline src/components/common/AppHeader.js:20]: perf

**Review by vtopham — APPROVED** [review]
Looks great!

**Pull request merged**